gateway-provider-ha/src/main/java/org/apache/knox/gateway/ha/dispatch/DefaultHaDispatch.java

```diff
@@ -39,7 +39,7 @@
 /**
  * Default HA dispatch class that has a very basic failover mechanism
  */
-public class DefaultHaDispatch extends DefaultDispatch {
+public class DefaultHaDispatch extends ConfigurableDispatch {
```

Comment:
This is a pretty drastic change. Should be separate from this.

Reply:
Agreed, but the change does not affect the existing features it just adds the configurability. So, in theory there is really no change. Creating a separate dispatch, like I mentioned in the PR, would simply duplicate the code.